packages/uhk-web/src/app/components/device/firmware/device-firmware.component.html

```diff
@@ -8,8 +8,12 @@ <h1>
             </h1>
 
             <p>
-                Firmware {{ hardwareModules.leftModuleInfo.firmwareVersion }} is running on the left keyboard half.<br>
-                Firmware {{ hardwareModules.rightModuleInfo.firmwareVersion }} is running on the right keyboard half.
+                <ng-container *ngFor="let moduleInfo of hardwareModules.moduleInfos; trackBy:moduleInfoTrackByFn">
```

Comment:
Please rather write:
Left keyboard half firmware: 8.10.7

Display the right keyboard half first.